indexers/query/AnnoyQueryIndexer/tests/test_annoy_query_indexer.py

```diff
@@ -0,0 +1,75 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import numpy as np
+import pytest
+from jina.executors.indexers.dump import export_dump_streaming
+from jina.executors.metas import get_default_metas
+
+from .. import AnnoyQueryIndexer
+
+# fix the seed here
+np.random.seed(500)
+retr_idx = None
+vec_idx = np.random.randint(0, high=100, size=[10]).astype(str)
+vec = np.random.random([10, 10])
+
+query = np.array(np.random.random([10, 10]), dtype=np.float32)
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+DUMP_DIR = os.path.join(cur_dir, 'dump')
+
+
+@pytest.fixture(scope='function', autouse=True)
+def metas(tmpdir):
+    metas = get_default_metas()
+    metas['workspace'] = tmpdir
+    metas['dump_path'] = os.path.join(tmpdir, 'dump')
+    yield metas
+
+
+def test_query_annoy_indexer_known_big(metas):
+    vectors = np.random.uniform(low=5.0, high=10.0, size=(10000, 1024))
+    # notice how it only reads from dump
+    queries = np.empty((10, 1024))
+    for idx in range(0, 10000, 1000):
+        array = idx * np.ones((1, 1024))
+        queries[int(idx / 1000)] = array
+        vectors[idx] = array
+
+    keys = np.arange(10000, 20000).astype(str)
+
+    export_dump_streaming(
+        path=metas['dump_path'],
+        shards=1,
+        size=len(keys),
+        data=zip(keys, vectors, [b'NOTUSED' for _ in range(len(keys))])
+    )
+
+    with AnnoyQueryIndexer(index_filename='not_used', metas=metas) as indexer:
```

Comment:
can this be done inside the `AnnoyQueryIndexer` '__init__` to call super().__init__(index_filename=None, *args, **kwargs)` or not used?

Reply:
done